bundles/org.eclipse.packagedrone.repo.manage.system/src/org/eclipse/packagedrone/repo/manage/system/internal/SystemServiceImpl.java

```diff
@@ -90,8 +93,10 @@ private static String discoverHostname ()
 
         if ( hostname == null )
         {
-            try ( Reader reader = new FileReader ( "/etc/hostname" ) )
```

Comment:
I am not sure this is a wise idea here. That file is a system specific file. So the system encoding should be used!?